index.js

```diff
@@ -213,6 +215,7 @@ class K8sExecutor extends Executor {
         this.serviceAccount = this.kubernetes.serviceAccount || 'default';
         this.dnsPolicy = this.kubernetes.dnsPolicy || 'ClusterFirst';
         this.automountServiceAccountToken = this.kubernetes.automountServiceAccountToken === 'true' || false;
+        this.terminationGracePeriodSeconds = this.kubernetes.terminationGracePeriodSeconds || 60;
```

Comment:
is `60` current default ? this value should be the default value k8s assigns in the absence of this setting.

Reply:
from what I see from https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/ it's `30s`